airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/FreeTag.tsx

```diff
@@ -0,0 +1,23 @@
+import { useIntl } from "react-intl";
+
+import { ReleaseStage } from "core/request/AirbyteClient";
+
+import styles from "./FreeTag.module.scss";
+import { useFreeConnectorProgram } from "./hooks/useFreeConnectorProgram";
+
+interface FreeTagProps {
+  releaseStage: ReleaseStage;
+}
+
+// A tag labeling a release stage pill as free. Defined here for easy reuse between the
+// two release stage pill implementations (which should likely be refactored!)
+export const FreeTag: React.FC<FreeTagProps> = ({ releaseStage }) => {
+  const { enrollmentStatusQuery } = useFreeConnectorProgram();
+  const { isEnrolled } = enrollmentStatusQuery.data || {};
+  const { formatMessage } = useIntl();
+  const freeStages: Array<ReleaseStage | undefined> = ["alpha", "beta"];
```

Comment:
```suggestion
  const freeStages: ReleaseStage[] = ["alpha", "beta"];
```

I can't imagine why we would want `undefined` in this array?

Reply:
Not sure I quite follow - the change I suggested passes my TS checker just fine (so does `["alpha", "beta"].includes(releaseStage)`, for that matter). What error are you seeing there? I don't see how the type of `isEnrolled`

Reply:
...none, now :flushed:. I certainly remember only adding that `| undefined` in response to some `Argument of type ReleaseStage | undefined is not assignable to parameter of type ReleaseStage` error, but now it even type-checks if I go back to the inline array of strings I tried initially. I feel like I'm taking crazy pills, but I'll take the cleaner code, I guess.